<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmConsumo</class>
 <widget class="QMainWindow" name="frmConsumo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>393</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Administraci√≥n de Consumo</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lblAreas">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Areas</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblAreas_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Consumos Registrados</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblDispo">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>8</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Dispositivos</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblConsumoWh">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>68</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Consumo (W/h)</string>
    </property>
   </widget>
   <widget class="QListWidget" name="lstAreas">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>131</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="tblRegistrado">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>331</width>
      <height>192</height>
     </rect>
    </property>
    <property name="showGrid">
     <bool>true</bool>
    </property>
    <property name="columnCount">
     <number>3</number>
    </property>
    <column/>
    <column/>
    <column/>
   </widget>
   <widget class="QComboBox" name="cmbDispositivos">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>28</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtConsumoWH">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>88</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAgregar">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>70</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Agregar</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblConsumo">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>360</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Consumo Total</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtTotal">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>360</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnProcesar">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>350</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Procesar</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>393</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
